installer.sh

```diff
@@ -66,6 +67,17 @@ get_os_info() {
     esac
 }
 
+check_for_rosetta() {
```

Comment:
I would consider inlining this check inside of `install_based_on_os`, right after 
```
        "Darwin")
            install_from_bin_package "wasp-macos-x86_64.tar.gz"
```